helm/trivy/values.yaml

```diff
@@ -0,0 +1,100 @@
+nameOverride: ""
+fullnameOverride: ""
+
+image:
+  registry: docker.io
+  repository: aquasec/trivy
+  tag: 0.13.0
+  pullPolicy: IfNotPresent
+
+replicaCount: 1
+
+persistence:
+  enabled: true
+  storageClass: ""
+  accessMode: ReadWriteOnce
+  size: 5Gi
+
+resources:
+  requests:
+    cpu: 200m
+    memory: 512Mi
+  limits:
+    cpu: 1
+    memory: 1Gi
+
+rbac:
+  create: true
+  pspEnabled: true
+
+podSecurityContext:
+  runAsUser: 1000
+  runAsNonRoot: true
+  fsGroup: 1000
+
+securityContext:
+  privileged: false
+  readOnlyRootFilesystem: true
+
+logLevel: info
+trivy:
+  ## cacheDir Trivy cache directory
+  cacheDir: "/home/scanner/.cache"
```

Comment:
I'd not expose it as Helm value. See my previous comment where this value is used.